components/automate-ui/src/app/modules/infra-proxy/infra-node-details/infra-node-details.component.html

```diff
@@ -0,0 +1,155 @@
+<div class="content-container">
+  <div class="container">
+    <main>
+      <chef-breadcrumbs>
+        <chef-breadcrumb [link]="['/infrastructure/chef-servers']">Chef Infra Servers</chef-breadcrumb>
+        <chef-breadcrumb [routerLink]="['/infrastructure/chef-servers', serverId]">Organizations</chef-breadcrumb>
+        <chef-breadcrumb [routerLink]="['/infrastructure/chef-servers', serverId, 'organizations', orgId]">Nodes</chef-breadcrumb>
+         {{InfraNode?.name}}
+      </chef-breadcrumbs>
+      <chef-page-header>
+        <chef-heading data-cy="infra-node-head">{{InfraNode?.name}}</chef-heading>
+        <div class="meta-box" *ngIf="!nodeDetailsLoading">
+          <div class="summary-body">
+            <table>
+              <tr>
+                <th>
+                  <p class="meta-head">NODE INFROMATION</p>
+                </th>
+              </tr>
+              <tr>
+                <th>
+                  Environment
+                </th>
+                <td data-cy="node-env">{{InfraNode?.environment}}</td>
+              </tr>
+              <tr>
+                <th>
+                  Policy Group
+                </th>
+                <td>{{InfraNode?.policy_group === '' ? '--' : InfraNode?.policy_group}}</td>
```

Comment:
You'd have to double check, but I believe you could simplify to this if you'd like.  Your call 👍 

```suggestion
                <td>{{InfraNode?.policy_group  ??  '--'}}</td>
```

Reply:
though it may only work with null/undefined and it appears this is looking for an empty string, so maybe not